src/dfmt/formatter.d

```diff
@@ -73,10 +73,13 @@ struct TokenFormatter(OutputRange)
 {
     /**
      * Params:
+     *     rawSource = ?
      *     tokens = the tokens to format
+     *     depths = ?
      *     output = the output range that the code will be formatted to
      *     astInformation = information about the AST used to inform formatting
      *         decisions.
+     *     config = ?
```

Comment:
Apologies for the question marks here. I don't yet know how to best describe these parameters.